src/popup/router/pages/Tip.vue

```diff
@@ -24,47 +24,17 @@
             {{ url }}
           </a>
         </template>
-        <Input
-          v-else
-          size="m-0 sm"
-          v-model="url"
-          :error="url && !validUrl"
-          :placeholder="$t('pages.tipPage.enterUrl')"
-        />
+        <Input v-else size="m-0 sm" v-model="url" :placeholder="$t('pages.tipPage.enterUrl')" />
       </div>
     </div>
     <div class="popup" data-cy="tip-container">
       <template v-if="!confirmMode">
-        <AmountSend
-          :amountError="amountError"
-          v-model="amount"
-          :errorMsg="
-            amount && !selectedToken && amount < minTipAmount
-              ? $t('pages.tipPage.minAmountError')
-              : ''
-          "
-        />
-        <Textarea
-          v-model="note"
-          :placeholder="$t('pages.tipPage.titlePlaceholder')"
-          :error="note.length > 280"
-          size="sm"
-        />
-        <Button
-          @click="toConfirm"
-          :disabled="
-            !note ||
-            amountError ||
-            noteError ||
-            !fee ||
-            !validUrl ||
-            !url ||
-            urlStatus === 'blacklisted' ||
-            note.length > 280
-          "
-          bold
-          data-cy="send-tip"
-        >
+        <AmountSend v-model="amount" />
+        <Textarea v-model="note" :placeholder="$t('pages.tipPage.titlePlaceholder')" size="sm" />
+        <div class="validation-msg" :class="{ show: validationStatus.msg }">
```

Comment:
```suggestion
        <div class="validation-msg">
```
With empty content it is anyway is not visible.